src/utils/date-time-helper.ts

```diff
@@ -57,4 +58,23 @@ export namespace DateTimeHelper {
 			return undefined;
 		}
 	}
+	export function checkWithinDays(value: string, withinDays: IWithinDaysRule) {
+		if (!value) return true;
+		const { numberOfDays, fromDate, dateFormat = "uuuu-MM-dd" } = withinDays;
+		const selectedDate = toLocalDateOrTime(value, dateFormat, "date");
+		if (!selectedDate) return false;
+		let startDate: LocalDate;
+		let endDate: LocalDate;
+		const baseDate = fromDate
+			? toLocalDateOrTime(fromDate, dateFormat, "date") || LocalDate.now()
+			: LocalDate.now();
+		if (numberOfDays >= 0) {
+			startDate = baseDate;
+			endDate = baseDate.plusDays(numberOfDays);
+		} else {
+			startDate = baseDate.plusDays(numberOfDays);
+			endDate = baseDate;
+		}
+		return !selectedDate.isBefore(startDate) && !selectedDate.isAfter(endDate);
```

Comment:
use truthy check for better readability
```suggestion
		return selectedDate.isAfter(startDate) && selectedDate.isBefore(endDate);
```